packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -78,16 +80,18 @@ const MessageContentWithLiveAria = (props: MessageContentWithLiveAriaProps): JSX
 const MessageContentAsRichTextHTML = (props: ChatMessageContentProps): JSX.Element => {
   /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
   useEffect(() => {
-    props.message.attachedFilesMetadata?.map((fileMetadata) => {
-      if (
-        props.onFetchAttachment &&
-        props.attachmentsMap &&
-        fileMetadata.attachmentType === 'inlineImage' &&
-        props.attachmentsMap[fileMetadata.id] === undefined
-      ) {
-        props.onFetchAttachment(fileMetadata);
-      }
+    const attachments = props.message.attachedFilesMetadata?.filter((fileMetadata) => {
+      return fileMetadata.attachmentType === 'inlineImage';
```

Comment:
keep in mind, future GA will just have attachmantType as `file` and `image`, determining if an image is inlineImage will be based on attachmentType == image AND content contains <img> that that has matching id in chat.content.attachments

Reply:
Yep we will make this refactor after we GA the new chat sdk with this change